packages/core/rpc/coreRpcMethods.ts

```diff
@@ -22,6 +23,27 @@ import SimpleFeature, { SimpleFeatureSerialized } from '../util/simpleFeature'
 export class CoreGetRefNames extends RpcMethodType {
   name = 'CoreGetRefNames'
 
+  async serializeArguments(args: RenderArgs, rpcDriverClassName: string) {
+    const assemblyManager = this.pluginManager.rootModel?.session
+      ?.assemblyManager
+
+    if (!assemblyManager) {
```

Comment:
can see here that the the fetching of the assembly config is done during serializeArguments for the rpc code.